static/templates/settings/account_settings.hbs

```diff
@@ -42,6 +42,13 @@
                             {{t 'Deactivate account' }}
                         </button>
                     </div>
+                    {{#if owner_is_only_user_organization}}
+                        <div id="deactivate_realm_personal_setting_container">
```

Comment:
I don't think we require this container element here. It was added for `Deactivate account` button such that tooltip can be shown when it is disabled.

Reply:
removed it.